src/java/us/kbase/mobu/initializer/test/DockerClientServerTester.java

```diff
@@ -166,9 +168,36 @@ protected File initPython(String moduleName) throws Exception {
         return moduleDir;
     }
     
+    public static void correctDockerfile(File moduleDir) throws Exception {
+        File buildXmlFile = new File("build.xml");
+        File sdkSubFolder = new File(moduleDir, "kb_sdk");
+        File sdkDistSubFolder = new File(sdkSubFolder, "dist");
+        FileUtils.copyFile(buildXmlFile, new File(sdkSubFolder, buildXmlFile.getName()));
```

Comment:
Does this create directories if missing? Or does /kb/module/kb_sdk/dist already exist (I dont think so?)?